CODEOWNERS

```diff
@@ -1 +1 @@
-@oss-integrations
+* @astronomer/oss-integrations
```

Comment:
Thanks for fixing this, @pankajkoti ! I hadn't realised before we were missing the org.